### ISSUE TITLE
[Accessibility]: Missing list element for upload list

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#44552]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

When navigating the list of uploads, there is no announcements of upload numbers there being a list element present.

### Expected behavior

There should be an audio cue indicating the list + element numbers.

### Additional details

https://user-images.githubusercontent.com/93103176/202476567-c0b0851f-ffb5-4444-a062-c5c9d831e6c0.mov

Similar issue: https://github.com/sourcegraph/sourcegraph/issues/35265

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
